src/style-config/z-index.scss

```diff
@@ -1,6 +1,6 @@
 $z-animate-menu-button:     310 !default;
 $z-dropdown-list:           10 !default;
-$z-datepicker:              10 !default;
+$z-datepicker:              1002 !default;
```

Comment:
does this still sit on top of the modal background? (it has a higher z-index)

Reply:
The datapicker does sit above the modal background yes. The background z-index is `1002` and the picker `1001` [https://github.com/Sage/carbon/blob/1260dfbc6da113b81112216c73cc580360b09c54/src/style-config/z-index.scss#L6]. Is this what you mean?